migrations/1671299400000-Deposit.ts

```diff
@@ -0,0 +1,16 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class Deposit1671299400000 implements MigrationInterface {
```

Comment:
Just curious: what's the significance of this timestamp?

`date -u -r 1671299400 
Sat Dec 17 17:50:00 UTC 2022`

Reply:
this is the timestamp when the migration file was created. The migration files contain the queries that define the structure of the database: creating tables, adding columns, altering tables, etc, so it's crucial for the migration files to run chronologically